data-prepper-plugins/otel-proto-common/build.gradle

```diff
@@ -15,6 +15,5 @@ dependencies {
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     implementation "org.apache.commons:commons-lang3:3.12.0"
     implementation 'commons-codec:commons-codec:1.15'
-    testImplementation "org.hamcrest:hamcrest:2.2"
```

Comment:
Same as above, Is this dependency not used?

Reply:
Same answer as above.